Misc/NEWS.d/next/Core_and_Builtins/2024-12-07-13-06-09.gh-issue-127599.tXCZb_.rst

```diff
@@ -0,0 +1 @@
+Fixes ``_Py_RefcntAdd`` missing calls to ``_Py_INCREF_STAT_INC`` / ``_Py_INCREF_IMMORTAL_STAT_INC``
```

Comment:
I'd suggest having a changelog a bit more general, e.g., "Fix statistics for an object's reference count is incremented (possibly by more than one in a single round)" or something like this (I don't have a good formulation for this so we can bikeshed a bit more). Unless we document those macros and `_Py_RefcntAdd`, I don't think it helps the user reading the changelog entry (AFAIK, `_Py_RefcntAdd` and the macros are only internal).

Reply:
I've pushed an alternative description that captures the more abstract information. Let me know if you think it's better.